css-view-transitions-1/Overview.bs

```diff
@@ -1368,11 +1371,15 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 			1. Let |capture| be a new [=captured element=] struct.
 
-			1. Set |capture|'s [=old image=] to the result of [=capturing the image=] of |element|.
+			1. Set |capture|'s [=should generate old pseudo=] to true.
 
 			1. Let |originalRect| be [=snapshot containing block=] if |element| is the [=document element=],
 				otherwise, the element|'s [=border box=].
 
+			1. Let |boundingBox| be |element|'s [=Element/get the bounding box|bounding box=], with [=ink overflow=] taken into account.
```

Comment:
nit: The spec uses "element’s ink overflow rectangle" here: https://drafts.csswg.org/css-view-transitions-1/#capture-the-image-algorithm:~:text=Return%20the%20portion%20of%20this%20canvas%20that%20includes%20element%E2%80%99s%20ink%20overflow%20rectangle. Use the same here for consistency?

And you had it right earlier that if this is the document element then |boundingBox| is the snapshot containing block. Might be simpler to write this as:

if element is document element, capture
else if element's ink overflow rect intersects snapshot containing block, capture.